compiler/pash_compiler.py

```diff
@@ -92,9 +93,17 @@ def compile_ir(ir_filename, compiled_script_file, args, compiler_config):
         ret = compile_optimize_output_script(
             ir_filename, compiled_script_file, args, compiler_config
         )
+    except ExpansionError as e: 
+        log("WARNING: Exception caught because some region(s) are not expandable and therefore unparallelizable:", e) 
+    except (ExpansionError, UnparallelizableError) as e: 
```

Comment:
Is `ExpansionError` here duplicated on purpose?

Reply:
My bad this should be fixed in the newest commit!